arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -41,7 +39,7 @@
         "tags": [
           "Namespaces"
         ],
-        "operationId": "Namespaces_ListBySubscription",
+        "operationId": "Namespaces_List",
```

Comment:
@sazeesha What I mean here is when you have new api-version `2017-xx-xx` for service bus that'd be the best time for these changes unless you are okay with existing SDKs having breaking changes.